.buildkite/pipeline.yml

```diff
@@ -9,6 +9,7 @@ steps:
             - path:
                 - auditbeat/
                 - .buildkite/auditbeat/
+                - .buildkite/hooks/
```

Comment:
additionally we should update the diff command for `auditbeat` and `heartbeat` to look like

`git diff --name-only origin/${GITHUB_PR_TARGET_BRANCH}...HEAD`

Reply:
Done!